packages/sdk/src/wallet/core/namespace/__tests__/WalletNamespace.spec.ts

```diff
@@ -97,7 +98,7 @@ describe('WalletNamespace', () => {
       const hostedWallet =
         await walletProvider.hostedWalletProvider.toVerbsWallet({
           walletId: privyWallet.id,
-          address: privyWallet.address,
+          address: privyWallet.address as Address,
```

Comment:
❓ Can we remove the need for this? If it's a hassle, we should take a full repo pass removing type casting later.

Reply:
type casts in tests arent the worst thing but we should definitely try to remove as many of them in non test files as possible